low-level/files/opt/seagate/sspl/setup/resource_map/lr2/resource_map.py

```diff
@@ -45,22 +45,6 @@ def get_health_info(self, rpath):
         """
         pass
 
-    @staticmethod
-    def get_node_details(node):
```

Comment:
why this common function moved out, resource map is common to both health & manifest !

Reply:
Corrected